diff --git a/monailabel/interfaces/app.py b/monailabel/interfaces/app.py
index 215f8c5..aaf4b4a 100644
--- a/monailabel/interfaces/app.py
+++ b/monailabel/interfaces/app.py
@@ -34,6 +34,7 @@ from monai.apps import download_and_extract
 from timeloop import Timeloop
 
 from monailabel.config import settings
+from monailabel.datastore.databricks_client import DatabricksClient
 from monailabel.datastore.dicom import DICOMwebClientX, DICOMWebDatastore
 from monailabel.datastore.dsa import DSADatastore
 from monailabel.datastore.local import LocalDatastore
@@ -160,6 +161,14 @@ class MONAILabelApp:
             logger.info("Creating DICOM Credentials for Google Cloud")
             dw_session = create_session_from_gcp_credentials()
             dw_client = DICOMwebClient(url=self.studies, session=dw_session)
+        elif "databricks.com" in self.studies:
+            logger.info("Creating DICOM Client for Databricks")
+            dw_client = DatabricksClient(
+                url=self.studies,
+                token=settings.MONAI_LABEL_DATABRICKS_TOKEN,
+                warehouse_id=settings.MONAI_LABEL_DATABRICKS_WAREHOUSE_ID,
+                table=settings.MONAI_LABEL_DATABRICKS_PIXELS_CATALOG_TABLE,
+            )
         else:
             if settings.MONAI_LABEL_DICOMWEB_USERNAME and settings.MONAI_LABEL_DICOMWEB_PASSWORD:
                 dw_session = create_session_from_user_pass(
